onvif/client.py

```diff
@@ -347,7 +347,9 @@ async def create_pullpoint_subscription(self):
         """Create a pullpoint subscription."""
         try:
             events = self.create_events_service()
-            pullpoint = await events.CreatePullPointSubscription()
+            pullpoint = await events.CreatePullPointSubscription(
+                {"InitialTerminationTime": "PT2H"}
```

Comment:
What is the motivation for choosing this value ?

Reply:
I don't know what a sensible default value should be, but if i don't explicit set the InitialTerminationTime, the subuscription failed (error 500).
Bugfix is developed by pierrebateau on Tapo C225 and also works for me with my C320WS.
I can test with different values, if you tell me what i have to insert as value. "PT1M"?

Reply:
Because onvif implementation vary widely in quality this change comes with the following risk

Cameras may not be able to handle relative dates, but this may be barter than an absolute date since some cameras will have the incorrect time set
Cameras may fail when passed an initial time
If we have a long subscription time hass may not retry to restore the subscription for 2 hours if the camera rebooted

But we get the following benefits:
Works will more cameras
We don't call renew as often which would improve performance for ones that use the 1M default

It's a difficult decision on if we should merge this or not because every onvif changes seems to cause collateral damage 

We need some more testing on other cameras with this change before we can proceed to a solution

Reply:
Initial testing with this change looks good

```
        termination_time = (
            (dt_util.utcnow() + dt.timedelta(days=1))
            .isoformat(timespec="seconds")
            .replace("+00:00", "Z")
        )
        with suppress(*SUBSCRIPTION_ERRORS):
            # The first time we renew, we may get a Fault error so we
            # suppress it. The subscription will be restarted in
            # async_restart later.
            await self._subscription.Renew(termination_time)
```

We do the renewals at 1d so we already have the case covered for < 1 day